### DISCUSSION TITLE
Battery of Aqara temperature sensor goes down and up and down and ...

### DISCUSSION THREAD

**Discussion by metawops** [Koenkk/zigbee2mqtt#15991] [Devices]
Today I stumbled upon some interesting historical data of one of my Aqara temperature sensors:

<img width="1462" alt="Screenshot 2023-01-06 at 20 13 07" src="https://user-images.githubusercontent.com/376090/211084109-3a5ee9b5-4fb0-4fcf-933a-5bd20e11f013.png">

How ca it be that the percentage rises again after falling? Several times? 😳

**Comment by evlo** [comment]
ambient temp.?
